packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx

```diff
@@ -287,12 +304,22 @@ const GraphicalRecording: React.FC = () => {
             onCancel={handleCancelSelection}
           />
           <Signature
-            visible={showSignature}
+            visible={showFirstSignature}
+            onCancel={handleCancelSelection}
+            winnerName={(selectedWinner && fullName(selectedWinner)) ?? null}
+            onConfirm={handleConfirmFirstSignature}
+            white={graphicalRecordingMode.pairing.players[0]}
+            black={graphicalRecordingMode.pairing.players[1]}
+            currentPlayer={graphicalRecordingMode.pairing.players[0]}
+          />
+          <Signature
+            visible={showSecondSignature}
             onCancel={handleCancelSelection}
             winnerName={(selectedWinner && fullName(selectedWinner)) ?? null}
             onConfirm={handleConfirmWinner}
-            whitePlayer={graphicalRecordingMode.pairing.players[0]}
-            blackPlayer={graphicalRecordingMode.pairing.players[1]}
+            white={graphicalRecordingMode.pairing.players[0]}
+            black={graphicalRecordingMode.pairing.players[1]}
+            currentPlayer={graphicalRecordingMode.pairing.players[1]}
           />
```

Comment:
Same deal here, I think we can clean this up by having only 1 signature component, where we change the current player to change who is signing